Common++/src/IpAddress.cpp

```diff
@@ -484,10 +488,10 @@ namespace pcpp
 	uint8_t IPv6Network::getPrefixLen() const
 	{
 		uint8_t result = 0;
-		for (auto byteIndex = 0; byteIndex < IPV6_ADDR_SIZE; byteIndex++)
+		for (const unsigned char byteIndex : m_Mask)
```

Comment:
We can rename `byteIndex` to `byte` and use `auto`:
```cpp
for (auto byte : m_Mask)
{
    ...
}
```

Reply:
fixed with 3ec344e61e6259128576e06d809e58ae62a9cc0b